src/api/collections/controller.ts

```diff
@@ -1,6 +1,7 @@
 import { cloudinary } from "@/services/cloudinary";
 import redis from "@/services/redis";
 import supabase from "@/services/supabase";
+import utils from "@/utils";
```

Comment:
well, no, if they forge the jwt, they will have to deal with the db because of no caching on Admins; db will return an error because of invalid signature